<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pretendo</class>
 <widget class="QMainWindow" name="Pretendo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pretendo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pretendo.qrc">
    <normaloff>:/pretendo/graphics/pretendo_32x32x8.png</normaloff>:/pretendo/graphics/pretendo_32x32x8.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QGLVideo" name="video" native="true">
          <property name="minimumSize">
           <size>
            <width>512</width>
            <height>480</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_ROM"/>
    <addaction name="action_Free_ROM"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_CPU">
    <property name="title">
     <string>&amp;CPU</string>
    </property>
    <addaction name="action_Run"/>
    <addaction name="action_Stop"/>
    <addaction name="action_Pause"/>
    <addaction name="separator"/>
    <addaction name="action_Hard_Reset"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShow_Sprites"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_CPU"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Load_ROM"/>
   <addaction name="action_Run"/>
   <addaction name="action_Pause"/>
   <addaction name="action_Stop"/>
  </widget>
  <action name="action_Load_ROM">
   <property name="icon">
    <iconset resource="pretendo.qrc">
     <normaloff>:/pretendo/graphics/pretendo_eject.png</normaloff>:/pretendo/graphics/pretendo_eject.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load ROM</string>
   </property>
  </action>
  <action name="action_Free_ROM">
   <property name="text">
    <string>&amp;Free ROM</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="icon">
    <iconset resource="pretendo.qrc">
     <normaloff>:/pretendo/graphics/pretendo_play.png</normaloff>:/pretendo/graphics/pretendo_play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="pretendo.qrc">
     <normaloff>:/pretendo/graphics/pretendo_stop.png</normaloff>:/pretendo/graphics/pretendo_stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="pretendo.qrc">
     <normaloff>:/pretendo/graphics/pretendo_pause.png</normaloff>:/pretendo/graphics/pretendo_pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionShow_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Background</string>
   </property>
  </action>
  <action name="actionShow_Sprites">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Sprites</string>
   </property>
  </action>
  <action name="action_Hard_Reset">
   <property name="text">
    <string>&amp;Hard Reset</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGLVideo</class>
   <extends>QWidget</extends>
   <header>QGLVideo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pretendo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>Pretendo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
